css-brochure-form/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h2 style="text-align: center">LEARN MORE</h2>
```

Comment:
You want to avoid inline styles at all cost. It's super bad practice. Inline styles have the 2nd highest specificity level, so it will override any css that you try to apply in your css files, so it makes this a lot harder to maintain, and your coworkers will hate you for it